deploy/samconfig.toml

```diff
@@ -104,3 +104,28 @@ s3_prefix = "patrickb-core-back-stack"
 region = "eu-west-2"
 capabilities = "CAPABILITY_IAM"
 parameter_overrides = "Environment=\"dev-patrickb\""
+
+[dev-vamh.deploy.parameters]
+stack_name = "ipv-core-back-dev-vamh"
+s3_bucket = "aws-sam-cli-managed-default-samclisourcebucket-ec647gpjuo2w"
+s3_prefix = "patrickb-core-back-stack"
```

Comment:
please update these prefix with the correct name.

Reply:
good catch (i thought the regex caught everything obviously not :) ) - should be good now.